charts/tests/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM bats/bats:1.9.0
+
```

Comment:
A lot of these tests apply equally for CP and VR, and there is a fair amount of duplication.  I think this is ok for now (this PR adds a huge amount of value and I don't want to block it), but maybe a task for the backlog to consider a more complicated test structure/framework that would allow us to commonise the test cases.